usaspending_api/api_docs/api_documentation/financial_spending/major_object_class.md

```diff
@@ -0,0 +1,62 @@
+
+## Financial Spending Major Object Class
+**Route**: `/api/v2/financial_spending/major_object_class?fiscal_year=[year]&funding_agency_id=[id]`
+
+Retrieve award obligation amounts broken down by object class.
```

Comment:
this needs to be: `Retrieve award obligation amounts broken down by major object class.`

Reply:
Done